shared/js/ui/base/ui-wrapper.js

```diff
@@ -23,7 +23,7 @@ export const openExtensionPage = (path) => {
     browser.tabs.create({ url: getExtensionURL(path) })
 }
 
-module.exports = {
+export default {
```

Comment:
Likewise for this one, do we need to do this here, or can we save for the follow-up?